imageio/plugins/tifffile.py

```diff
@@ -63,7 +63,8 @@ def load_lib():
 
 
 class TiffFormat(Format):
-    """Provides support for a wide range of Tiff images.
+    """Provides support for a wide range of Tiff images using the tifffile
+    plugin.
```

Comment:
```suggestion
    backend.
```

Using "plugin" here may be a bit confusing. We happen to refer to the `Format` sub-classes interchangably as plugins or formats; hence, this can be seen as a circular definition. Using "tifffile backend" should make this more clear.